editor/components/inner-blocks/index.js

```diff
@@ -3,8 +3,8 @@
  */
 import { withContext } from '@wordpress/components';
 
-function InnerBlocks( { BlockList, layouts, allowedBlocks, template } ) {
-	return <BlockList { ...{ layouts, allowedBlocks, template } } />;
+function InnerBlocks( { BlockList, layouts, allowedBlocks, locking, template } ) {
+	return <BlockList { ...{ layouts, allowedBlocks, locking, template } } />;
```

Comment:
See previous comments on naming.

Reply:
I changed the prop to "lock", in templates we use template_lock so I think using "lock" here is consistent. Unfortunately using isLocked in this case is not possible because we are not setting a boolean value.

Reply:
>I changed the prop to "lock", in templates we use template_lock, so I think using "lock" here is consistent. Unfortunately using isLocked in this case is not possible because we are not setting a boolean value.

But in the context of InnerBlocks, isn't the locking still referring to the template?